### PR TITLE
refactor: dashboard, drawers, upcoming interviews

### PR DESCRIPTION
**Pull request by angelstackit**

### CODE REVIEW

File: src/layout/DashboardMainContainer.tsx [hunk 1]
```diff
@@ -1,25 +1,22 @@
 import StaffOnly from '@/components/auth/staffOnly'
 import { BigCalendar } from '@/components/BigCalendar/BigCalendar'
-import { useAppSelector } from '@/store/hooks/useAppSelector'
+// import { useAppSelector } from '@/store/hooks/useAppSelector'
```
Comment (juliswer) [inline src/layout/DashboardMainContainer.tsx:3]: please remove comments

**Review by juliswer — DISMISSED** [review]
Looks good. Please check my comments before merging

File: src/layout/DashboardMainContainer.tsx (hunk 1 above)
Comment (angelstackit) [reply src/layout/DashboardMainContainer.tsx]: Solved Julian-senpai

**Review by Naiby — APPROVED** [review]
OSOM

**Pull request merged**